.github/workflows/docs_changes_summary.yml

```diff
@@ -0,0 +1,69 @@
+name: Summarize changes to source plugins docs
+
+on:
+  # Using pull_request_target works on forked PRs too. This is safe since we don't checkout the PR code (we only use the diff)
+  pull_request_target:
+    branches:
+      - main
+
+jobs:
+  doc-changes:
+    defaults:
+      run:
+        working-directory: scripts/table_diff
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

Comment:
Naive question: does this not check out the PR code when used on `pull_request_target`?

Reply:
`pull_request_target` runes in the context of the `main` branch so this checks out the `main` branch code.
In order to checkout the PR's code, one needs to pass the PR `sha` explicitly by doing:
```
- uses: actions/checkout@v2
      with:
        ref: ${{ github.event.pull_request.head.sha }}
```

more in https://securitylab.github.com/research/github-actions-preventing-pwn-requests/

Reply:
Ah okay, got it, thanks for the explanation!